test/fast/Unit tests/nvm_default_packages

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+die () { echo "$@" ; cleanup ; exit 1; }
+cleanup () {
+  rm -rf "$(nvm_version_path v6.10.1)" ../../../default-packages
+}
+
+cat > ../../../default-packages << EOF
+gulp
+bower@1.7.9
+
+# commented-package
+
+webpack/webpack
```

Comment:
let's pick some smaller packages for the tests (than webpack or gulp) and still-relevant ones (unlike bower) - how about `rimraf`, `object-inspect`, things like that?